bots/tests-invoke

```diff
@@ -235,6 +246,20 @@ class PullTask(object):
         sys.stderr.write(msg)
 
         ret = None
+
+        offline = ('offline' in opts) and opts.offline
+        if not ret and self.base and "TEST_INVOKE_RESPAWN" not in os.environ:
+            ret = ret or self.rebase(offline)
```

Comment:
The `ret or` is redundant, as the `if` already ensures that it's None (which is also redundant), and there's nothing between line 248 and here that would set it. It's harmless, but I wonder if you intended something else here?